xp0-double/xp0d-azuredeploy-msdeploy.json

```diff
@@ -116,18 +116,6 @@
       "minLength": 8,
       "defaultValue": "[concat(toUpper(uniqueString('cd-web')), '@', uniqueString(parameters('sqlserver_password')))]"
     },
-    "analytics_mongodb_connectionstring": {
-      "type": "securestring"
-    },
```

Comment:
It's only using the `Tidy` versions (search the file for uses to verify), e.g. 

`"analyticsMongoDbConnStrTidy": "[parameters('provisioningOutput').analyticsMongoDbConnStrTidy.value]",`

It seems to be getting all of this data from the provisioning settings, and it's not erroring with the removal of that property.